modules/generate/generate.go

```diff
@@ -8,27 +8,66 @@ package generate
 import (
 	"crypto/rand"
 	"encoding/base64"
+	"errors"
+	"github.com/dgrijalva/jwt-go"
 	"io"
 	"math/big"
 	"time"
-
-	"github.com/dgrijalva/jwt-go"
 )
 
 // GetRandomString generate random string by specify chars.
 func GetRandomString(n int) (string, error) {
-	const alphanum = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+	const lower = "abcdefghijklmnopqrstuvwxyz"
+	const upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	const digit = "0123456789"
+	const spec = "!@#$%^&*()_-+=[]'\";:/?.>,<`~"
```

Comment:
This is a breaking change for this function, some functions that use GetRandomString, may not expect special characters

Reply:
Fixed. Added new function GetRandomPassword.